src/app/modules/deployments/components/secret-management-detail/secret-management-detail.component.ts

```diff
@@ -205,12 +205,22 @@ export class SecretManagementDetailComponent implements OnInit {
             });
     }
 
-    secretNameValid(): boolean {
-        const name = this.secretFormGroup.get('secret')?.getRawValue();
+    secretNameIsUnique(): boolean {
+        let valid = true;
+        const name = this.secretFormGroup.get('secret')?.getRawValue().trim();
         if (this.secrets.find((s) => s.name === name)) {
-            return false;
+            valid = false;
         }
-        return true;
+        return valid;
+    }
+
+    secretNamehasNoWhitespaces(): boolean {
+        let valid = true;
+        const name = this.secretFormGroup.get('secret')?.getRawValue().trim();
+        if (name.length === 0) {
+            valid = false;
+        }
+        return valid;
```

Comment:
Valid variable can be avoided if using
return name.length > 0;

Reply:
Also the function name can be a little misleading, you may think that white spaces are not allowed. Maybe is better something like: secretNameIsNotJustWhitespaces or something like that.

Reply:
Perfect thank you! I already fixed both issues